api/can_api_definition.py

```diff
@@ -72,12 +72,20 @@ class Metrics(base_model.APIBaseModel):
 
     testPositivity: Optional[float] = pydantic.Field(
         ...,
-        description="Ratio of people who test positive calculated using a 7 day rolling average.",
+        description="Ratio of people who test positive calculated using a 7-day rolling average.",
     )
 
     caseDensity: Optional[float] = pydantic.Field(
         ...,
-        description="The number of cases per 100k population calculated using a 7 day rolling average.",
+        description="The number of cases per 100k population calculated using a 7-day rolling average.",
+    )
+
+    contactTracerCapacity: Optional[float] = pydantic.Field(
```

Comment:
Nit on naming though the ship may have sailed.... capacity makes me think the value is an absolute count rather than ratio. contactTracerCaseRatio?